backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/modal/BpkModal.kt

```diff
@@ -72,8 +74,12 @@ fun BpkModal(
         val activityWindow = getActivityWindow()
         val dialogWindow = getDialogWindow()
         val parentView = LocalView.current.parent as View
+        val isSystemInDarkTheme = isSystemInDarkTheme()
         SideEffect {
             if (activityWindow != null && dialogWindow != null) {
+                if (!isSystemInDarkTheme) {
```

Comment:
couldn't find that solution earlier but turns out i was just looking at the wrong window and have to set it for the dialog window (dark mode is correct, so only need it for light mode)